cluster/common.sh

```diff
@@ -278,6 +290,10 @@ function tars_from_version() {
     echo "Failure trying to curl Salt tar .sha1"
   done
 
+  if ! curl -Ss --range 0-1 "${KUBE_TAR_URL}" >&/dev/null; then
```

Comment:
note: existence checks on gcs should use --head. This pattern was not introduced in this pr so we don't have to fix it here.

https://cloud.google.com/storage/docs/xml-api/head-object?hl=en